packages/forklift-console-plugin/src/plans/create/steps/GeneralInformationForm.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+import { FormErrorHelperText } from 'src/components/FormErrorHelperText';
+import { useForkliftTranslation } from 'src/utils';
+
+import {
+  Flex,
+  Form,
+  FormGroup,
+  FormSection,
+  TextInput,
+  Title,
+  ValidatedOptions,
+} from '@patternfly/react-core';
+
+import { GeneralFormFieldId, generalFormFieldLabels } from './constants';
+
+export const GeneralInformationForm: FC = () => {
+  const { t } = useForkliftTranslation();
+  const {
+    control,
+    formState: { errors },
+  } = useFormContext();
+
+  return (
+    <Flex
+      direction={{ default: 'column' }}
+      spaceItems={{ default: 'spaceItemsLg' }}
+      className="create-plan-wizard__form"
+    >
+      <Title headingLevel="h2">{t('General')}</Title>
+
+      <Form>
+        <FormSection title={t('Plan information')} titleElement="h3">
+          <p>{t('Name your plan and choose the project you would like it to be created in.')}</p>
+
+          <FormGroup
+            isRequired
+            fieldId={GeneralFormFieldId.PlanName}
+            label={generalFormFieldLabels[GeneralFormFieldId.PlanName]}
+          >
+            <Controller
+              name={GeneralFormFieldId.PlanName}
+              control={control}
+              rules={{
+                required: t('Plan name is required.'),
+              }}
+              render={({ field }) => (
+                <TextInput
+                  {...field}
+                  validated={
+                    errors[GeneralFormFieldId.PlanName]
```

Comment:
do we need it? isnt this triggered automatically when next step is clicked?

Reply:
Yes, this `validated` prop is what determined the text input's red outline. On "next" click, or any trigger of validation, the `errors` will be updated to reflect any new entry, and the ID is what we're using to point to the correct error value here - to again - determine whether the input should have a red outline or not.

Reply:
i think if u add
              error={Boolean(fieldState?.error)}
fieldState , is the second argument u can deconstruct, {field, fieldState}, we will get the same effect without the need for ternary check

Reply:
btw i think u can use               helperText={fieldState?.error?.message}
and omit the formerrorhelpertext , please try

Reply:
Add `error={Boolean(fieldState?.error)}` where exactly? This is not a prop that PF's `TextInput` has, nor `FormGroup`. And when I print fieldState inside of that callback for the render function, I just see an empty object even when an error is triggered.

Also, `helperText` is not a prop that `TextInput` or `FormGroup` have.

Patternfly's example of using error helper text can be seen in their demos, which shows a similar implementation to what we have here: 
https://www.patternfly.org/components/helper-text/react-demos#static-text-and-dynamic-status